examples/jsm/lines/LineMaterial.js

```diff
@@ -324,6 +322,38 @@ class LineMaterial extends ShaderMaterial {
 
 			},
 
+			dashed: {
+
+				enumerable: true,
+
+				get: function () {
+
+					return Boolean( 'USE_DASH' in this.defines );
+
+				},
+
+				set( value ) {
+
+					if ( Boolean( value ) !== Boolean( 'USE_DASH' in this.defines ) ) {
+
+						this.needsUpdate = true;
+
+					}
+
+					if ( value ) {
```

Comment:
Maybe this should be more robust?

```js
if ( value === true ) {
```

Reply:
Done! Change the alpha to coverage setter, too.